scripts/prepare-patch-branch

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -e
+
+if [[ $# -lt 1 ]]; then
+    echo 'Please provide a title for your patch branch' >&2
+    exit 1
+fi
+PATCH_TITLE=$1
+
+for REMOTE in $(git remote); do
+    URL=$(git remote get-url $REMOTE)
+    if [[ $URL =~ "turt2live" ]]; then
+        UPSTREAM_REPO=$REMOTE
+    elif [[ $URL =~ "vector-im" ]]; then
+        FORK_REPO=$REMOTE
+    fi
+done
+
+function echoAndDo {
```

Comment:
Can this be done via https://stackoverflow.com/a/2853811?

Reply:
Not quite; the top suggestion of `set -v` prints every line, which effectively prints the whole script file.

What I went with is more like this other suggestion further down: https://stackoverflow.com/a/23342259